src/uncrustify.cpp

```diff
@@ -451,7 +454,6 @@ int main(int argc, char *argv[])
 
    if (arg.Present("--show-config"))
    {
-      set_option_defaults();
```

Comment:
(Note: Not needed! Defaults are set by the initializers :slightly_smiling_face:.)